src/layouts/BlogPost.astro

```diff
@@ -1,13 +1,15 @@
 ---
 import type { CollectionEntry } from "astro:content";
-import BaseHead from "../components/AstroCompoent/base/BaseHead.astro";
-import Header from "../components/AstroCompoent/header/Header.astro";
-import Footer from "../components/AstroCompoent/footer/Footer.astro";
-import FormattedDate from "../components/FormattedDate.astro";
+
+import BaseHead from "@astro/base/BaseHead.astro";
+import Header from "@astro/header/Header.astro";
+import Footer from "@astro/footer/Footer.astro";
+
+import FormattedDate from "@components/FormattedDate.astro";
 import { Image } from "astro:assets";
-import ShareButtons from "../components/ReaactCompoents/SocialButton/socialButton.tsx";
+import ShareButtons from "@react/SocialButton/socialButton.tsx";
 import SpeedInsights from "@vercel/speed-insights/astro";
-import PostComment from "../components/AstroCompoent/BlogComments/PostComment.astro";
+import PostComment from "@astro/BlogComments/PostComment.astro";
```

Comment:
I have told you many time, I am not changing the name of the folder, it will cause alot of issues, 

So Leave  it. and all the paths are defined in the astro config and ts config file so you will see these path will work.